### ISSUE TITLE
MD036 sees emphasized paragraphs as header.

### ISSUE THREAD

**Issue by astorije** [markdownlint/markdownlint#140]
In a document, I have the following:

``` markdown
[Text]

**Caution: This will destroy all local changes to these folders so make sure to
commit and push, or squash them before running.**

[Text]
```

`markdownlint` will mark add the following in its report: `MD036 Emphasis used instead of a header`.

In this case, this is a full sentence, starting with a capital, ending with a period, and containing punctuations. It also has 2 lines. Visually, this is not a header, it is a paragraph.

Would it be possible to treat any content with trailing punctuation (similar to what [MD026](https://github.com/mivok/markdownlint/blob/master/docs/RULES.md#md026---trailing-punctuation-in-header) does) and/or multiple lines as non-headers?

**Comment by mivok** [comment]
I like both suggested fixes here. It should be possible to implement them to reduce false positives on MD036.
